src/controllers/users.controller.js

```
@@ -0,0 +1,28 @@
+import { usersService } from '../services/users.service.js';
+import { normalize } from '../utils/validate.js';
+
+const getAll = async (req, res) => {
+  const users = await usersService.getAllActive();
+
+  if (!users) {
+    return res.status(400);
```

Error: The getAll endpoint returns only a status code (res.status(400)) without a response body. This is not standard and may cause issues for clients expecting a JSON response. You should return a JSON error message, e.g., 'return res.status(400).json({ message: "No users found" });'.